code/datums/memory/_memory.dm

```diff
@@ -75,6 +75,25 @@
 	memorizer_mind = null
 	return ..()
 
+/datum/memory/serialize_list(list/options)
+	. = list()
+	.["name"] = name
+	if(memorizer)
+		.["memorizer"] = memorizer
+	.["story_value"] = story_value
+	.["memory_flags"] = memory_flags
+	if(mood_verb)
+		.["mood_verb"] = mood_verb
+	if(protagonist_name)
+		.["protagonist_name"] = protagonist_name
+	if(deuteragonist_name)
+		.["deuteragonist_name"] = deuteragonist_name
+	if(antagonist_name)
+		.["antagonist_name"] = antagonist_name
+	if(where)
+		.["where"] = where
+	options[SCHEMA_VERSION] = "1.0"
```

Comment:
This isn't semver. Semver is 1.0.0.

Reply:
The patch is important in case we need to do something like fix a property coming out as null or something.

Reply:
Make sure to assert that SCHEMA_VERSION is valid semver (in our case, just major.minor.patch)